assets/src/application/services/context/test/data.ts

```diff
@@ -0,0 +1,29 @@
+import { GET_CURRENT_USER } from '@dataServices/apollo/queries/currentUser';
+import { GET_GENERAL_SETTINGS } from '@dataServices/apollo/queries/generalSettings';
+import { mockEspressoDomData } from '../../config/test/data';
+import { MockedResponse } from '@edtrServices/context/TestContext/types';
```

Comment:
This may need some changes in barista because it imports from domain, although it's OK for tests, but I have tried to separate the things in barista.